cmd/bots/bots.go

```diff
@@ -3,15 +3,14 @@ package main
 import (
```

Comment:
Why did you move this file? I think every app should have own directory. Create new directory for telegram bot.